packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -74,6 +83,11 @@ export class SignIn extends BaseResource implements SignInResource {
   prepareFirstFactor = (factor: PrepareFirstFactorParams): Promise<SignInResource> => {
     let config;
     switch (factor.strategy) {
+      // @ts-ignore As this is experimental we want to support it at runtime, but not at the type level
```

Comment:
I'd like to know what the rest of the team thinks about this because in my opinion we should ship and use these experimental types. The name is clear enough and would allow us to not loose TS support while developing this.

The methods are also public so why not also make the types public?

Reply:
1. They will become public once the feature is out of the experimental stage.
2. I wanted to "hide" them as these will affect the current public and stable APIs we have like `prepareFirstFactor`.